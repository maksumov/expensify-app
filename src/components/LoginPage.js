import React, { Component } from 'react'

export const LoginPage = () => (
  <div>
    <button>Login</button>
  </div>
)
