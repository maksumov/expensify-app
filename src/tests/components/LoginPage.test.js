import React from 'react'
import { shallow } from 'enzyme'
import LoginPage from '../../components/LoginPage'

test('should render LoginPage component', () => {
  const wrapper = shallow(<LoginPage />)
  expect(wrapper).toMatchSnapshot()
})
