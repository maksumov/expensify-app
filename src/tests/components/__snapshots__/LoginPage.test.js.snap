// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render LoginPage 1`] = `
<div>
  <button
    onClick={[MockFunction]}
  >
    Login
  </button>
</div>
`;
